cypress/e2e/tests/custom-metrics/assessment_metrics.test.ts

```diff
@@ -0,0 +1,72 @@
+import {
+    createMultipleApplications,
+    createMultipleStakeholders,
+    deleteAllStakeholders,
+    deleteApplicationTableRows,
+    login,
+} from "../../../utils/utils";
+import { Assessment } from "../../models/migration/applicationinventory/assessment";
+import { Metrics } from "../../models/migration/custom-metrics/custom-metrics";
+import { Stakeholders } from "../../models/migration/controls/stakeholders";
+
+let applicationList: Array<Assessment> = [];
+let metrics = new Metrics();
+let stakeholdersList: Array<Stakeholders> = [];
+
+describe(["@tier2"], "Custom Metrics - The total number of initiated assessments", function () {
+    before("Login and create test data", function () {
+        // Perform login
+        login();
+
+        // Navigate to Application inventory tab, delete all applications
+        deleteApplicationTableRows();
+
+        // Navigate to stakeholders control tab and create new stakeholder
+        stakeholdersList = createMultipleStakeholders(1);
+
+        // Create 2 applications
+        applicationList = createMultipleApplications(2);
+    });
+
+    after("Perform test data clean up", function () {
```

Comment:
Let's move "after "after all the tests .

Reply:
Done.